server/sonar-web/src/main/js/apps/projects/filters/Filter.js

```diff
@@ -40,19 +40,25 @@ export default class Filter extends React.PureComponent {
     getFacetValueForOption: React.PropTypes.func,
 
     halfWidth: React.PropTypes.bool,
-    highlightUnder: React.PropTypes.func
+    highlightUnder: React.PropTypes.any
```

Comment:
I think I can just put `number`. I was using `any` because I wanted a number or the default boolean, but in the end I just check if the value exists so I don't need the default boolean value.